.github/workflows/pr-dev.yml

```diff
@@ -0,0 +1,47 @@
+name: DEV
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint-and-test:
+    name: Lint and Unit Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Python ${{ matrix.node-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.node-version }}
+      - run: python -m unittest
+      - run: python -m pip install pylint
+      - run: python -m pylint --rcfile=.pylintrc *.py **/*.py
```

Comment:
This is probably going to fail without dependancies having been installed. Please refer to Dockerfile. There are a few strategies for doing this:

- install global using pipenv
- create requirements.txt then install from there.
- run with pipenv

I recommend going with the requirements.txt strategy, as that is going to give you the cleanest install closest to production.